Python/compile.c

```diff
@@ -4015,8 +4066,10 @@ compiler_visit_stmt(struct compiler *c, stmt_ty s)
                 n++;
             }
         }
-        ADDOP_I(c, RAISE_VARARGS, (int)n);
+        location loc = LOC(s);
```

Comment:
There's a few cases of this pattern:
```C
    location loc = LOC(s);
    ADDOP...(..., loc, ...);
```
Any reason not to shorten this to:
```C
    ADDOP...(..., LOC(S), ...);
```
?

Reply:
I'll land this and do this and other cleanup in followup PRs. (There are still the SET_LOC and UNSET_LOC to remove - they do nothing now). 

Thanks for the review!